.github/ISSUE_TEMPLATE/joinin.md

```diff
@@ -0,0 +1,23 @@
+Welcome to OpenLab!
+
+Here are some ways to support OpenLab:
```

Comment:
maybe need a more detailed introduction about to becoming a member of OpenLab teams, "some ways to support OpenLab" is too broad description. the can also include donation, integration testing...